# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem
# Commit: 55e22304ccae
# Full Hash: 55e22304ccaea2af33219077000171821441822c
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2025-08-07 04:23:39
# Description:
#   Bug 1981271 - reset state specific to the built candidate chain root when continuing path building r=nkulatova
#   
#   Any state that NSSCertDBTrustDomain keeps with regard to a built candidate
#   chain (and in particular, the root of that chain), must be reset if path
#   building continues with other potential chains. In particular, this means that
# ==============================================================================

diff -r 2c224e8a6135 -r 55e22304ccae security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/security/manager/ssl/tests/unit/test_trust_anchors/int-by-ca2.pem	Wed Aug 06 18:13:58 2025 +0000
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
+MIICyzCCAbOgAwIBAgIUS6A/IqoBn7U01u9RsbapALzHFMMwDQYJKoZIhvcNAQEL
+BQAwDjEMMAoGA1UEAwwDY2EyMCIYDzIwMjMxMTI4MDAwMDAwWhgPMjAyNjAyMDUw
+MDAwMDBaMA4xDDAKBgNVBAMMA2ludDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+AQoCggEBALqIUahEjhbWQf1utogGNhA9PBPZ6uQ1SrTs9WhXbCR7wcclqODYH72x
+nAabbhqG8mvir1p1a2pkcQh6pVqnRYf3HNUknAJ+zUP8HmnQOCApk6sgw0nk27lM
+wmtsDu0Vgg/xfq1pGrHTAjqLKkHup3DgDw2N/WYLK7AkkqR9uYhheZCxV5A90jvF
+4LhIH6g304hD7ycW2FW3ZlqqfgKQLzp7EIAGJMwcbJetlmFbt+KWEsB1MaMMkd20
+yvf8rR0l0wnvuRcOp2jhs3svIm9p47SKlWEd7ibWJZ2rkQhONsscJAQsvxaLL+Xx
+j5kXMbiz/kkj+nJRxDHVA6zaGAo17Y0CAwEAAaMdMBswDAYDVR0TBAUwAwEB/zAL
+BgNVHQ8EBAMCAQYwDQYJKoZIhvcNAQELBQADggEBAGk01PNPhBLMqkPax52xAwbA
+SfFv/5yxc5CbfY9SJHp6usCPTjwso7N12XrEW9CsJH6+QGaWMjA/l13tvtmxRAev
+7CbnS3Wd/giBYkPXzwkwlVIHjdp/dZoNHFY1V7yhkVmX/GRlxMOMrOkH7iGw5a0C
+pW8snqmrNK4fjIPJ6y2kkrz+QF0LJHR+L6u1dPnsAup5YqoqyZyB6McTqXThaN++
+IQEL+GLCcfdy8PztYAHZY3tHQZYneV5c3UHK3SSQqz8/S5q7q+7OVsg8DFxp1UDP
+OHpS+QZa/zkgiasdvO7gfvp+R1aD0QvOmMHX2M/VaL8/GJDA5wg+Ty9i3sPE7yw=
+-----END CERTIFICATE-----